data/gui/window/game_version.cfg

```diff
@@ -1,165 +1,441 @@
+					id = "tab_label"
+					wrap = true
+				[/label]
+				
+			[/column]
+			
+			{_GUI_VERINFO_TAB_PADDING}
+			
+		[/row]
+		
+	[/grid]
+	
+[/toggle_panel]
+#enddef
 
+#define _GUI_VERINFO_TAB_PAGE_GENERAL
+	[row]
 		[column]
-			grow_factor = 0
-
-			border = "all"
-			border_size = 5
-			horizontal_grow = true
-
-			[button]
-				id = "browse_"+{ID_STEM}
-				definition = "action_browse"
-				label = _ "filesystem^Browse"
-				tooltip = _ "Browse this location using a file manager"
-			[/button]
-
+			vertical_alignment = top
+			horizontal_alignment = left
+			[grid]
+				[row]
+					[column]
+						border = "left,bottom"
+						border_size = 5
+						horizontal_alignment = left
+						[grid]
+							[row]
+								[column]
+									horizontal_alignment = left
+									[label]
+										label = _ "✦ <b>Version:</b>"
+										use_markup = yes
+									[/label]
+								[/column]
+								[column]
+									grow_factor = 0
+									horizontal_alignment = left
+									border = "left"
+									border_size = 5
+									
+									[label]
+										id = version
+									[/label]
+								[/column]
+							[/row]
+						[/grid]
+					[/column]
+				[/row]
+				[row]
+					[column]
+						border = "left,bottom"
+						border_size = 5
+						horizontal_alignment = left
+						[grid]
+							[row]
+								[column]
+									horizontal_alignment = left
+									[label]
+										label = _ "✦ <b>Running on:</b>"
+										use_markup = yes
+									[/label]
+								[/column]
+								[column]
+									grow_factor = 0
+									horizontal_alignment = left
+									border = "left"
+									border_size = 5
+									
+									[label]
+										id = os
+										use_markup = yes
+									[/label]
+								[/column]
+							[/row]
+						[/grid]
+					[/column]
+				[/row]
+				[row]
+					[column]
+						border = "left,bottom"
+						border_size = 5
+						horizontal_alignment = left
+						[grid]
+							[row]
+								[column]
+									horizontal_alignment = left
+									[label]
+										label = _ "✦ <b>Architecture:</b>"
+										use_markup = yes
+									[/label]
+								[/column]
+								[column]
+									grow_factor = 0
+									horizontal_alignment = left
+									border = "left"
+									border_size = 5
+									
+									[label]
+										id = arch
+									[/label]
+								[/column]
+							[/row]
+						[/grid]
+					[/column]
+				[/row]
+				[row]
+					[column]
+						border = "left,bottom"
+						border_size = 5
+						horizontal_alignment = left
+						[grid]
+							[row]
+								[column]
+									horizontal_alignment = left
+									[label]
+										label = _ "✦ <b>Homepage:</b>"
+										use_markup = yes
+									[/label]
+								[/column]
+								[column]
+									grow_factor = 0
+									horizontal_alignment = left
+									border = "left"
+									border_size = 5
+									
+									[label]
+										label = _ "<span foreground='yellow'><u>https://www.wesnoth.org</u></span>"
```

Comment:
Sorry for not noticing this sooner, but does a link really need to be translated? Same applies for the link to the wiki below.

Reply:
Resolved in #8928.